### DISCUSSION TITLE
Upgrade to node 18 or 20?

### DISCUSSION THREAD

**Discussion by GoaMind** [n4bb12/verdaccio-github-oauth-ui#187] [General]
Good day,

Is there any plan to upgrade this awesome plugin to Node 18 or Node 20?
Thank you in advance. 🙇 

Regards,
Evgeny

**Comment by n4bb12** [comment]
Hi Evgeny, eventually yes. Thanks for bringing it up.

I believe, someone reported issues with Node 18 when it first came out. Considering that 20 is already the current LTS, we should definitely update.

I will see if I can spot any problems with 18 and 20.

Maybe it would also be good to remove the engines part from the package json and only document versions that are known to work. That way, we don't prevent anyone from using other versions of node that work for them.

**Comment by n4bb12** [comment]
You can now use Node 18 and 20.
https://github.com/n4bb12/verdaccio-github-oauth-ui/releases/tag/6.0.3
